## class``异常处理`

`
# aert

# assert 1 + 1 == 3
# print(3) # if there is a False behind assert,the program will stop and output `AssertionError`

# you can add a parameter to assert like `assert (temp >= 0), "在0以下！"` 
# the willing output is `AssertionError: 在0以下！`
```
<!--stackedit_data:
eyJoaXN0b3J5IjpbLTEyODE1NjgxODddfQ==
-->